docs/prow_plugins_additional_tooling.md

```diff
@@ -0,0 +1,234 @@
+The milestone plugin allows members of a configurable GitHub team to set the milestone on an issue or pull request.
+
+**milestonestatus**
+
+The milestonestatus plugin allows members of the milestone maintainers Github team to specify the 'status/*' label that should apply to a pull request.
+
+**needs-rebase**
+The needs-rebase plugin manages the 'needs-rebase' label by removing it from Pull Requests that are mergeable and adding it to those which are not.
+
+**override**
+
+The override plugin allows repo admins to force a github status context to pass.
+
+**owners-label**
+
+The owners-label plugin automatically adds labels to PRs based on the files they touch.
+
+**release-note**
+
+The releasenote plugin implements a release note process that uses a markdown 'releasenote' code block to associate a release note with a pull request.
+
+**require-matching-label**
+
+The require-matching-label plugin is a configurable plugin that applies a label to issues and/or PRs that do not have any labels matching a regular expression.
+
+**require-sig**
+
+When a new issue is opened the require-sig plugin adds the "needs-sig" label and leaves a comment requesting that a SIG (Special Interest Group) label be added to the issue.
+
+**shrug**
+
+¯\_(ツ)_/¯
+
+**sigmention**
+
+The sigmention plugin responds to SIG (Special Interest Group) Github team mentions like '@kubernetes/sig-testing-bugs'.
+
+**size**
+
+The size plugin manages the 'size/*' labels, maintaining the appropriate label on each pull request as it is updated.
+
+**skip**
+
+The skip plugin allows users to clean up Github stale commit statuses for non-blocking jobs on a PR.
+
+**slackevents**
+
+The slackevents plugin reacts to various Github events by commenting in Slack channels.
+
+**stage**
+
+Label the stage of an issue as alpha/beta/stable.
+
+**trigger**
+
+The trigger plugin starts tests in reaction to commands and pull request events.
+
+**verify-owners**
+
+The verify-owners plugin validates OWNERS files if they are modified in a PR.
+
+**welcome**
+
+The welcome plugin posts a welcoming message when it detects a user's first contribution to a repo.
+
+**wip**
+
+The wip (Work In Progress) plugin applies the 'do-not-merge/work-in-progress' label to pull requests whose title starts with 'WIP' and removes it from pull requests when they remove the title prefix.
+
+**yuks**
+
+The yuks plugin comments with jokes in response to the `/joke` command.
+
+#### Plugins we will use for the first installation
+- approve
+- assign
+- blunderbuss
+- config-updater
+- docs-no-retest
+- help
+- hold
+- label
+- lgtm
+- owners-label
+- size
+- trigger
+- verify-owners
+- wip
+
+#### Plugins that can be integrated later on demand
+- blockade
+- cat
+- cherry-pick-unapproved
+- dog
+- lifecycle
+- needs-rebase
+- override
+- owners-label
+- release-note
+- require-matching-label
+- require-sig
+- shrug
+- sigmention
+- skip
+- slackevents
+- welcome
+- yuks
+
+### Additional Tooling used with Prow[2]
```

Comment:
I recommend to change order: 
1. Tooling which we are using
2. Tooling to check